openapi/components/schemas/GatewayAccountConfig/Stripe.yaml

```
@@ -40,6 +40,15 @@ allOf:
           copyCredentialsFrom:
             type: string
             description: ID of an existing Stripe gateway account from which credentials are copied in order to skip the onboarding process.
+          setupFutureUsage:
+            type: string
+            description: |-
+              Indicates intent to make future payments with this payment method.
+              on_session: Reuse the payment method when the customer is present.
+              off_session: Reuse the payment method when the customer may or may not be present.
```

suggestion: I may help readability to add these as bullets 

I think they'll be rendered as a paragraph otherwise.

Also, it may be good to highlight the variable names

```suggestion
              - `on_session`: Reuse the payment method when the customer is present.
              - `off_session`: Reuse the payment method when the customer may or may not be present.
```